src/api/types/mesh.c

```diff
@@ -229,6 +229,20 @@ int l_lovrMeshSetMaterial(lua_State* L) {
   return 0;
 }
 
+int l_lovrMeshAttachAttributes(lua_State* L) {
+  Mesh* attachTo = luax_checktype(L, 1, Mesh);
+  Mesh* attachThis = luax_checktype(L, 2, Mesh);
+  int instanceDivisor = luaL_optnumber(L, 3, 0);
+  // TODO: Check attribute name(s) in 4th argument and if present only attach those
+
+  VertexFormat* format = &attachThis->vertexData->format;
```

Comment:
I try to not directly access stuff from objects in the API code (unless they are in the data module since they're basically just raw structs).  Here, it would be possible to use `lovrMeshGetVertexFormat`.